exercises/binary/src/example.c

```diff
@@ -0,0 +1,23 @@
+int len(char *str)
+{
+   int count = 0;
+   while (*str++) {
+      count++;
+   }
+   return count;
+}
+
+int convert(char *input)
+{
+   int input_len = len(input);
```

Comment:
@wolf99  it is a part of my conversion so i thought i should implement myself , its not my interpretation it is given in exercism official website?..  , Anyways i will add strlen() in the next commit..